app/views/my/access_requests/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= content_for :heading do %>
+  Access Requests
+<% end %>
+
+<h2>Member Registry Access</h2>
+
+<article>
+  <p>This page details requests to view the register of association members.</p>
+
+  <p>Access to the register by members is legally required - our policy outlines how requests are made and the manner in which access is granted.</p>
```

Comment:
(when #104 is merged) - please link `our policy`

Reply:
I have that change stashed locally, waiting for this moment :)